torchvision/prototype/transforms/_transform.py

```diff
@@ -83,3 +86,85 @@ def forward(self, *inputs: Any) -> Any:
         ]
 
         return tree_unflatten(flat_outputs, spec)
+
+
+class _DetectionBatchTransform(Transform):
```

Comment:
I've opted to put this common functionality into a superclass that all other detection batch transforms can subclass from. Maybe we can even improve this further, but I'll get to that when I fix `SimpleCopyPaste`.

Reply:
Ah Thanks! This is much better! 👍